### PR TITLE
Feat - WIP 327 - About background color

### PR DESCRIPTION
**Pull request by jl-ib**
#### What does this PR do?
- Fixes the background color of the ABOUT button when active, also adds an underline when it is active as visual feedback for the user.

#### Where should the reviewer start?
App.css file

#### How should this be manually tested?
1. Clicking the about button once logged in

#### What are the relevant tickets?
[https://wizeline.atlassian.net/browse/WIP-327](https://wizeline.atlassian.net/browse/WIP-327)

#### Screenshots
image

### CODE REVIEW

File: src/styles/App.css [hunk 1]
```diff
@@ -24,7 +24,13 @@ h2 {
   line-height: 5.5vh !important;
 }
 
-.md-text.center-vertical.active > a {
+.md-text.center-vertical.active > a{
+  color: #777 !important;
```
Comment (isa-bel) [inline src/styles/App.css]: Never use `!important`, please!

File: src/styles/App.css [hunk 2]
```diff
@@ -24,7 +24,13 @@ h2 {
...
+  background-color: #111111 !important;
+}
+
+.navbar-default .navbar-nav>.active>a{
```
Comment (isa-bel) [inline src/styles/App.css]: Spacing

**Comment by arrutneo** [comment]
Agree with @isa-bel, if the rules does not work without `!important` please make the CSS selector more specific. Maybe with another level of hierarchy or with a class name. Thanks!

File: src/styles/App.css (hunk 1 above)
Comment (jl-ib) [inline src/styles/App.css]: Thanks Isabel 😄 , I made the changes.

File: src/styles/App.css [hunk 3]
```diff
@@ -24,9 +24,18 @@ h2 {
   line-height: 5.5vh !important;
 }
 
-.md-text.center-vertical.active > a {
-  color: #777 !important;
-  background-color: #111111 !important;
+.md-text.center-vertical.active > 
+.navbar-default .navbar-nav>.active>a,
```
Comment (isa-bel) [inline src/styles/App.css]: Could you add spaces around `>` ? 
To make it clearer.
`navbar-default .navbar-nav > .active > a,`

File: src/styles/App.css [hunk 4]
```diff
@@ -24,9 +24,18 @@ h2 {
...
-  background-color: #111111 !important;
+.md-text.center-vertical.active > a,
+.navbar-default .navbar-nav > .active > a,
+.navbar-default .navbar-nav > .active > a:focus,
+.navbar-default .navbar-nav > .active > a:hover {
+  color: #777;
+  background-color: #111111;
+}
+
+.navbar-default .navbar-nav>.active>a {
```
Comment (isa-bel) [inline src/styles/App.css]: Spaces here also please.

**Pull request merged**